.github/workflows/ci.yml

```diff
@@ -1,9 +1,9 @@
 name: CI
 on:
   push:
-    branches: [ "master", "v0.7.x", "v0.8.x" ]
```

Comment:
Can you undo all these formatting changes? It's hard to see what has actually changed.